content/en/Methodologies/ai-llm.md

```diff
@@ -6,17 +6,17 @@ description: >
   Review Cobalt pentest methodologies for an AI/LLM application.
 ---
 
-At Cobalt, we follow an industry-standard methodology based on the [OWASP Top 10 for Large Language Model Applications]([https://genai.owasp.org/llm-top-10/](https://owasp.org/www-project-top-10-for-large-language-model-applications/)). 
+Cobalt follows an industry-standard methodology based on the [OWASP Top 10 for Large Language Model Applications]([https://genai.owasp.org/llm-top-10/](https://owasp.org/www-project-top-10-for-large-language-model-applications/)). 
 
-Cobalt offers two levels of Artificial Intelligence (AI) and Large Language Model (LLM) pentesting of Web and Web + API Assets.
+Cobalt offers two levels of Artificial Intelligence (AI) and Large Language Model (LLM) pentesting for Web and Web + API Assets.
 
 - [LLM/AI Prompt Injection](#llm-prompt-injection) (+4 credits)
-Focus on testing the security of your AI systems against prompt injection attacks. These attacks manipulate the AI’s input to generate malicious output, which can compromise the system’s integrity and confidentiality. Prompt Injection AI/LLM pentests are run as an Agile pentest with an [automated report](/platform-deep-dive/pentests/reports/#pentest-report-types).
+Cobalt focused on testing AI systems security against prompt injection attacks. These attacks manipulate the AI’s input to generate malicious output, which can compromise the system’s integrity and confidentiality. Prompt Injection AI/LLM pentests are run as an Agile pentest with an [automated report](/platform-deep-dive/pentests/reports/#pentest-report-types).
 
 - [Full Coverage LLM](#llm) (+16 credits)
```

Comment:
Same with this link

Reply:
removed these - must have been leftover - thanks.